### ISSUE TITLE
Horizontal scrolling

### ISSUE THREAD

**Issue by adriannelee** [cds-snc/notification-planning#590]
Issue group: Tables
Visibility: 2
Evaluation: Partially Supports
Success Criteria: 1.4.10: Reflow
Needs design: 

### Description
On mobile layouts the “Your profile” page may require horizontal scrolling depending on the length of the Email address. This can be remedied by allowing the text of the email address to break/wrap at any point.

![](https://dl.airtable.com/.attachments/5e3a427d27459b556a7664b7168a44c1/fe64a153/Capturedecranle2022-03-15a13_29_33.png)

**Comment by andrewleith** [comment]
@adriannelee, I am unable to re-produce this.  Even with a long email address, it appears to wrap fine.  Is it possible this has been fixed? I checked with my phone too (android) and things wrapped there as well.

![image](https://github.com/cds-snc/notification-planning/assets/823749/90af536c-adb3-4655-a4e3-6c6516266df0)

**Comment by adriannelee** [comment]
@andrewleith My guess is that the auditor couldn't test this himself last year and mentioned it in case it might be an issue. I think we can move this to Done. What do you think @amazingphilippe ?

**Comment by amazingphilippe** [comment]
@andrewleith I think you migth be zoomed out. The overflow might not create a scrollbar. If you look at the header and footer, they seem to be narrower than your current viewport, because they are being pushed out by the width of the table.

**Comment by andrewleith** [comment]
Hmm you are right, I was zoomed out!  Also, when I test this again it is not breaking/wrapping like in my screenshot.  Not sure what was going on there!

So, if we update the css a bit we can get this to do some word breaking, but it isn't perfect.  Here is using `overflow-wrap: anywhere` on `.do-not-truncate-text`:

![image](https://github.com/cds-snc/notification-planning/assets/823749/dd6b8bd4-87b4-45a8-ae23-0a5254c060f3)

@amazingphilippe is this a good solution?  Or should we try something else?

**Comment by amazingphilippe** [comment]
Its something! It fixes the horizontal scrolling, but breaks readability. So its not a good trade-off. Maybe we should spend some time to design table-less solution. I don'T think this is the best use case for a table anyways. A flexbox layout would work really nicely.

**Comment by andrewleith** [comment]
Yeah makes sense.  I spent a little time on this Friday, I'll post a PR in a few!

**Comment by andrewleith** [comment]
Pushed a PR yesterday to address this layout: https://github.com/cds-snc/notification-admin/issues/1684

**Comment by andrewleith** [comment]
@amazingphilippe could you take a look when you have a chance?

**Comment by whabanks** [comment]
QA'd in production, looks good!
